experimental/README.md

```diff
@@ -9,12 +9,35 @@ issues associated with it. If necessary, links are provided to additional
 documentation on an issue.  As an active Docker user and community member,
 please feel free to provide any feedback on these features you wish.
 
-## Install Docker experimental
+## Use Docker experimental
 
-Unlike the regular Docker binary, the experimental channels is built and
-updated nightly on https://experimental.docker.com. From one day to the
-next, new features may appear, while existing experimental features may be
-refined or entirely removed.
+Experimental features are now included in the standard Docker binaries as of
+version 1.13.0.
+For enabling experimental features, you need to start the Docker daemon with
+`--experimental` flag.
+You can also enable the daemon flag via `/etc/docker/daemon.json`. e.g.
+
+        {
+            "experimental": true
+        }
+
+Then make sure the experimental flag is enabled:
+
+        $ docker version -f '{{.Server.Experimental}}'
+        true
+
+## Install from `experimental.docker.com`
+
+If you install Docker from `experimental.docker.com`, the experimental features
+are enabled by default.
+The experimental package available at `experimental.docker.com` is almost
+identical to the standard Docker package, but it contains `daemon.json` for
```

Comment:
https://github.com/docker/docker/issues/28194#issuecomment-267133263
> There will not be a release of experimental binaries for the 1.13.0 GA, so no need to create work for making sure the binaries are released.

Should we just remove this subsection?
PTAL @andrewhsu